README.md

```diff
@@ -1,23 +1,24 @@
 # DevOps 101
 
-What is Continuous Integration?
-What is Continuous Delivery?
+The objective of the `DevOps 101` is to explain how you can create a CICD Pipeline on IBM Cloud in order to automate full life cycle management of your application.
```

Comment:
Make sure to put a `/` between the CI and CD in all instances here, and it would be useful to mention what the acronym stands for. I think a brief 2-3 sentence paragraph differentiating the two would serve us well here too.